test/integration/patients/patient/patient-flow.js

```diff
@@ -1282,9 +1282,7 @@ context('patient flow page', function() {
       .routeFlowActions(fx => {
         fx.data = testFlowActions;
 
-        fx.included.push(
-          getForm({ id: '11111', attributes: { name: 'Test Form' } }),
-        );
+        fx.included.push(testForm);
```

Comment:
I think this could be removed.. we don't include forms on this endpoint

Reply:
Removed 👋